manifest_template.yml

```diff
@@ -80,18 +80,36 @@ apigee:
         attributes:
           - name: access
             value: public
-          - name: ratelimit
-            value: {{ ENV.ratelimit | default('5ps') }}
+          - name: ratelimiting
+            value:
+              e-referrals-service-api-{{ VARIANT.service | default(VARIANT.name) }}-{{ ENV.name }}:
+                quota:
+                  enabled: false
+                spikeArrest:
+                  enabled: false
+              app:
+                quota:
+                  enabled: true
+                  interval: 8
+                  limit: {{ ENV.requestLimit | default('288000') }}
+                  timeunit: hour
+                spikeArrest:
+                  enabled: true
+                  ratelimit: {{ ENV.burstLimit | default('720pm') }}
+          - name: requestLimit_binary
+            value: 57600
```

Comment:
I've set them constant for all environments to avoid any confusion. In the future we can refactor these and use dfferent rates across different environments.